timeseries/src/autogluon/timeseries/models/abstract/abstract_timeseries_model.py

```diff
@@ -217,9 +217,9 @@ def __init__(
         )
         self.val_score: Optional[float] = None  # Score with eval_metric (Validation data)
 
-        self.target_scaler: Optional[TargetScaler]
-        self.covariate_scaler: Optional[CovariateScaler]
-        self.covariate_regressor: Optional[CovariateRegressor]
+        self.target_scaler: Optional[TargetScaler] = None
+        self.covariate_scaler: Optional[CovariateScaler] = None
+        self.covariate_regressor: Optional[CovariateRegressor] = None
```

Comment:
We either need to set these to `None` here or inside `MultiWindowBacktestingModel._initialize_transforms_and_regressor`. Otherwise the if-condition `if self.target_scaler is not None` will raise an exception during `fit()` and `predict()`.

Reply:
If it only affects the MultiWindowBacktestingModel, let's initialize them there.

Reply:
Done